homeworks/olena.ambitska_elena-ambitska/GamesProject/5-logic/calc.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
I've meant the full integration without actually `5-logic` folder. The html file is near all other html files, css - to style folder, js - to srcipt folder, probably.
Ok, I'm merging, you can fix in the next task.

Reply:
@vladyslav-yermolin-moc  thanks. I will fix this in the next PR